manager/assets/modext/core/modx.js

```diff
@@ -911,6 +911,7 @@ Ext.extend(MODx.Msg,Ext.Component,{
               this.addListener(i,l.fn,l.scope || this,l.options || {});
             }
         }
+        Ext.MessageBox.minWidth = (config.minWidth) ? config.minWidth : 200;
```

Comment:
200 is not enough, you should set the size bigger, it seems to me

Reply:
It is the minWidth and not the width. So it does not have to be larger.

Reply:
I understand that it is minWidth. Here, for example, is how it looks at 200px with a window of 1920px (fits only 2 buttons, see gif).
In my opinion, the size is not enough even for minWidth, isn't it?

![trash](https://user-images.githubusercontent.com/12523676/52404292-1aac8480-2ae2-11e9-8f17-40af3a3fd420.gif)

Reply:
And line 903 becomes unnecessary, or am I mistaken?
`Ext.MessageBox.minWidth = 200;`

Reply:
Can we finish this conversation and resolve it, please?

Reply:
Line 903 is required for all other MessageBox types except "confirm".